packages/jwt/src/types.ts

```diff
@@ -0,0 +1,13 @@
+export type Algorithms = 'ES256' | 'ES256K';
```

Comment:
I still think we should change it from `ES256K` to something else. Maybe `EIP191` is the best as it describes what was performed: https://eips.ethereum.org/EIPS/eip-191
```suggestion
export type Algorithms = 'ES256' | 'EIP191';
```